ChangeLog

```diff
@@ -65,6 +65,7 @@
  * #1190 (computeProbability clamped by Domain-SmallVolume)
  * #1202 (Sample::sort & Sample::sortAccordingToAComponent only return new Samples)
  * #1216 (The CalibrationResult doc does not match the code)
+ * #1236 (Adding minimum volume set examples)
```

Comment:
should be `#997 (The computeMinimumVolumeLevelSet method has doc issues)`, see issues/997

Reply:
You are right: 53442fc fixes this.